document/core/exec/runtime.rst

```diff
@@ -804,7 +805,7 @@ Finally, the following definition of *evaluation context* and associated structu
    \begin{array}{llll}
    \production{(evaluation contexts)} & E &::=&
      [\_] ~|~
-     \val^\ast~E~\instr^\ast ~|~
+     val^\ast~E~\instr^\ast ~|~
```

Comment:
I removed `\` from all the `var_{...}` in the example because there `var` is part of a variable name (could have used anything, like `x`, `y`, `z` , as opposed to `\var`which is a  [non-terminal symbol]. However using the non-terminal symbols as variable names is widely used in the spec, so I guess it might be confusing to change in this example. Putting all the `\` back before merging.

Reply:
Writing "var" in TeX math mode is the same as "v a r", i.e., three variables. among other things, the macros ensure that it's correctly type-set as "\mathit{var}".